ui/redux/selectors/blocked.js

```diff
@@ -27,6 +28,6 @@ export const selectMutedAndBlockedChannelIds = createSelector(
       } catch {}
     });
 
-    return Array.from(uniqueSet).sort();
+    return Container.Array.useStableEmpty(Array.from(uniqueSet).sort());
```

Comment:
Array is sort of a reserved word, right? EmptyArr, EmptyObj?

Reply:
Actually, just Arr and Obj?

Reply:
Ok, will use Arr and Obj ... wanted something shorter anyway.